libs/dh/wholesale/data-access-api/src/lib/dh-wholesale-data-access-api.store.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { ComponentStore, tapResponse } from '@ngrx/component-store';
+import { filter, map, Observable, switchMap, tap } from 'rxjs';
```

Comment:
suggestion: remove this line. All of the imports are not used anywhere in the file.

Reply:
Actually I've made a mistake. Only `filter`, `map` and `tap` are not used. They can be removed

Reply:
Thx. Removed.